### PR TITLE
fix: colors in OpenAPI spec docs

### LINKED ISSUE #196: OpenAPI docs - change JSON strings color (contrast is not readable)

### PR DESCRIPTION
**Pull request by exu**
## Pull request description 

closes #196 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to TestKube - your somewhat opinionated and friendly Kubernetes testing framework!
 
-TestKube decouples test artefacts and execution from CI/CD tooling; tests are meant to be part of your
+TestKube decouples test artifacts and execution from CI/CD tooling; tests are meant to be part of your
 clusters state and can be executed as needed:
```
Comment (fog1985) [inline docs/index.md:6]: @exu , maybe there should not be a new line? I mean merge next with the present. Seems like the lines break is not intentional.

Comment (exu) [reply docs/index.md:6]: 👍🏻

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Known vulnerabilities: [![TestKube](https://snyk.io/test/github/kubeshop/testkub
                                                            
 # Welcome to TestKube - your friendly Kubernetes testing framework!
 
-TestKube decouples test artefacts and execution from CI/CD tooling; tests are meant to be part of your
+TestKube decouples test artifacts and execution from CI/CD tooling; tests are meant to be part of your
```
Comment (fog1985) [inline README.md:13]: @exu , the same here. Unnecessary line break.

**Review by nicufk — APPROVED** [review]

**Review by fog1985 — APPROVED** [review]
LGTM

**Pull request merged**